static/app/views/performance/transactionSummary/pageLayout.tsx

```diff
@@ -110,6 +113,28 @@ function PageLayout(props: Props) {
 
   const {isInDomainView} = useDomainViewFilters();
 
+  const dataCategories: [DataCategory, ...DataCategory[]] = useMemo(() => {
+    switch (tab) {
+      case Tab.PROFILING:
+        return [DataCategory.PROFILE_DURATION, DataCategory.PROFILE_DURATION_UI];
+      case Tab.REPLAYS:
+        return [DataCategory.REPLAYS];
+      case Tab.EVENTS:
+      case Tab.TAGS:
+      case Tab.WEB_VITALS:
+        return [DataCategory.TRANSACTIONS];
+      case Tab.TRANSACTION_SUMMARY:
+        return [DataCategory.SPANS];
```

Comment:
```suggestion
        return [DataCategory.SPANS, DataCategory.TRANSACTIONS];
```
for now. I see events-stats queries to `metricsEnhanced` and queries to the tags endpoint (which can soon be replaced by span attributes distribution)